### DISCUSSION TITLE
Possible ways to enable ghost atom exchanges in deepmd lammps simulation?

### DISCUSSION THREAD

**Discussion by qchempku2017** [deepmodeling/deepmd-kit#4520] [Q&A] [answered]
Dear community,

Recently I've been learning to use deepmd to perform thermodynamics sampling in lammps. My project involves sampling the configurational disorder between lattice atoms and vacancy defects. 

Currently, I am thinking about performing a mixed MC-MD sampling to do this task.  **I'm planning to add a special type of "ghost" atoms** with zero mass and do not interact with the rest of the system, while the interactions between the other atoms are calculated in deepmd pair_style. These ghost atoms will be fixed immobile during MD run between MC steps, but allowed to swap with other lattice-occupying atoms at MC steps, so that we can sample vacancy configurations. 

However, since I am relatively new to the lammps and Deepmd techniques, I am not sure if this "dummy atom" idea can be implemented. Can anyone show some hint? Much appreciated!

**Comment by njzjz** [comment, marked as answer]
See #2732; also documentation.
